### PR TITLE
Updates language assets for dutch

### PR DESCRIPTION
**Pull request by halfak**

### CODE REVIEW

**Comment by halfak** [comment]
@catrope, could you take a look at this for me?

File: revscoring/languages/dutch.py [hunk 1]
```diff
@@ -42,62 +42,99 @@
...
+    "balhaar",  # ball hair (testicular hair)
+    "debiel",  # infirm
+    "diaree", "diarree",  # diarrhea
+    "drol", "drollen",  # turd
+    "fack", "facking", "focking",  # misspelling of "fuck"
+    "flikker", "flikkers",  # perjorative for gay person ("faggot")
+    "geil", "geile",  # horny
+    "gelul",  # bullshit
+    "hoer", "hoere", "hoeren",  # whore
+    "homo", "homos",  # add "homo's" ; perjorative for gay person
```
Comment (catrope) [inline revscoring/languages/dutch.py]: There's an unresolved TODO here for adding "homo's"

File: revscoring/languages/dutch.py [hunk 2]
```diff
@@ -42,62 +42,99 @@
...
+    "kak", "kaka",  # poop
+    "kakhoofd", "kakken",  # kakhoofd = poopy head; kakken = to poop (verb)
+    "kanker", "kenker",  # cancer
+    "klootzak", "klootzakken",  # "ball sack"
+    "klote",  # lit.: balls; equivalent: "sucky"
+    "kolere", "klere",  # Chollera
```
Comment (catrope) [inline revscoring/languages/dutch.py]: "Cholera" is spelled with one "l"

File: revscoring/languages/dutch.py [hunk 3]
```diff
@@ -42,62 +42,99 @@
...
+    "kont", "kontgat",  # butt, butthole
+    "kontje",  # little butt
+    "lekkerding", "lekker ding",  # means something like "hot piece"
+    "likken",  # lick (not totally sure why this is here)
+    "pedo",  # add "pedofiel"; pedophile
```
Comment (catrope) [inline revscoring/languages/dutch.py]: Unresolved TODO

File: revscoring/languages/dutch.py [hunk 4]
```diff
@@ -42,62 +42,99 @@
...
+    "kak", "kaka",  # poop
+    "kakhoofd", "kakken",  # kakhoofd = poopy head; kakken = to poop (verb)
+    "kanker", "kenker",  # cancer
+    "klootzak", "klootzakken",  # "ball sack"
+    "klote",  # lit.: balls; equivalent: "sucky"
+    "kolere", "klere",  # Chollera
+    "kont", "kontgat",  # butt, butthole
+    "kontje",  # little butt
+    "lekkerding", "lekker ding",  # means something like "hot piece"
+    "likken",  # lick (not totally sure why this is here)
```
Comment (catrope) [inline revscoring/languages/dutch.py]: "not totally sure why this is here"

File: revscoring/languages/dutch.py [hunk 5]
```diff
@@ -42,62 +42,99 @@
...
+    "porn", "porno",  # porn
+    "neuk", "neuke", "neuken", "neukende", "neukt",  # "fuck" conjugations
+    "neukte", "neukten", "geneukt",  # "fuck" conjugations continued
+    "nicht", "nichten",  # "faggot" but also sometimes "cousin"
+    "strond", "stront",  # shit
+    "zuigt", "suckt",  # sucks
+    "sukkel", "sukkels",  # sucker (idiot)
+    "tering",  # colloquial word for tuberculosis, now a swear word;
+    "tiet", "tetten", "tieten",  # tits
+    "verekte", "verkracht", "verrekte",  # "damn" or "fucking" (adj)
```
Comment (catrope) [inline revscoring/languages/dutch.py]: "verkracht" is duplicated (should be removed on this line, stay on the next)

File: revscoring/languages/dutch.py [hunk 6]
```diff
@@ -42,62 +42,99 @@
...
+    "verkracht", "verkrachten",  # rape/raped
+    "dikzak",  # fat person
+    "mogolen", "mogool", "mongool", "mongolen",  # perj. for down syndrome
+    "mooiboy",  # man who puts a lot of effort into his appearance
+    "sperma",  # sperm
+    "kut", "kutje", "kutjes",  # vulgar word for vagina (Eng.: cunt)
+    "stelletje",  # "bunch of", as part of a racial slur or perj.
+    "lul",  # dick
+    "lullen",  # out of an ass
```
Comment (catrope) [inline revscoring/languages/dutch.py]: talking out of one's ass (since this is a verb)

File: revscoring/languages/dutch.py [hunk 7]
```diff
@@ -42,62 +42,99 @@
...
+    "smeerlap",  # literally: "grease rag"
+    "het zuigt",  # "It sucks"
+    "sukkel",  # "Sucker"
+    "sul",  # "wimp", "dork", or "schlemiel". Its etymology is unclear
+    "vreten",  # rude form of the verb "to eat"
+    "vuil", "vuile",  # "filth" or "filthy"
+    "wijf", "kutwijf", "kankerhoer", "rothoer", "vishoer",  # perj for women
+
+    # Racial slurs
+    "bamivreter",  # "bami eater" an ethnic slur used against people of Asian
```
Comment (catrope) [inline revscoring/languages/dutch.py]: add "descent" after "Asian"

File: revscoring/languages/dutch.py (hunk 1 above)
Comment (halfak) [reply revscoring/languages/dutch.py]: Aha!  thanks.

File: revscoring/languages/dutch.py (hunk 2 above)
Comment (halfak) [reply revscoring/languages/dutch.py]: Heh.  Thanks.

File: revscoring/languages/dutch.py (hunk 4 above)
Comment (halfak) [reply revscoring/languages/dutch.py]: That one shows up often in vandalism.  We extracted it with a script that compares words added in edits that get reverted but not in edits that don't get reverted.

**Comment by halfak** [comment]
OK @Catrope.  One more look?

**Pull request merged**